protocols/CreateExternSamplesProjects.sh

```diff
@@ -68,36 +67,36 @@ for ((samplenumber = 0; samplenumber <= max_index; samplenumber++))
 do
 	if [[ ${seqType[samplenumber]} == "SR" ]]
 	then
-  		if [[ ${barcode[samplenumber]} == "None" ]]
+		if [[ ${barcode[samplenumber]} == "None" ]]
 		then
-			ln -sf ../../../../../rawdata/ngs/${sequencingStartDate[samplenumber]}_${sequencer[samplenumber]}_${run[samplenumber]}_${flowcell[samplenumber]}/${externalFastQ_1[samplenumber]} ${projectRawtmpDataDir}/${sequencingStartDate[samplenumber]}_${sequencer[samplenumber]}_${run[samplenumber]}_${flowcell[samplenumber]}_L${lane[samplenumber]}_${barcode[samplenumber]}.fq.gz
-			ln -sf ../../../../../rawdata/ngs/${sequencingStartDate[samplenumber]}_${sequencer[samplenumber]}_${run[samplenumber]}_${flowcell[samplenumber]}/${externalFastQ_1[samplenumber]}.md5 ${projectRawtmpDataDir}/${sequencingStartDate[samplenumber]}_${sequencer[samplenumber]}_${run[samplenumber]}_${flowcell[samplenumber]}_L${lane[samplenumber]}_${barcode[samplenumber]}.fq.gz.md5
-  		else
-      			ln -sf ../../../../../rawdata/ngs/${sequencingStartDate[samplenumber]}_${sequencer[samplenumber]}_${run[samplenumber]}_${flowcell[samplenumber]}/${externalFastQ_1[samplenumber]} ${projectRawtmpDataDir}/${sequencingStartDate[samplenumber]}_${sequencer[samplenumber]}_${run[samplenumber]}_${flowcell[samplenumber]}_L${lane[samplenumber]}_${barcode[samplenumber]}.fq.gz
-      			ln -sf ../../../../../rawdata/ngs/${sequencingStartDate[samplenumber]}_${sequencer[samplenumber]}_${run[samplenumber]}_${flowcell[samplenumber]}/${externalFastQ_1[samplenumber]}.md5 ${projectRawtmpDataDir}/${sequencingStartDate[samplenumber]}_${sequencer[samplenumber]}_${run[samplenumber]}_${flowcell[samplenumber]}_L${lane[samplenumber]}_${barcode[samplenumber]}.fq.gz.md5
+			ln -sf ${externalFastQ_1[samplenumber]} ${projectRawtmpDataDir}/${sequencingStartDate[samplenumber]}_${sequencer[samplenumber]}_${run[samplenumber]}_${flowcell[samplenumber]}_L${lane[samplenumber]}_${barcode[samplenumber]}.fq.gz
```

Comment:
Waarom worden de relatieve paden hier omzeep geholpen? Als je nu de resultaten verplekt van tmp0* naar prm0*, dan zijn de links kappot...

Reply:
Dit gaat om externe samples, waarbij on het makkelijker vind om de volledige paden te gebruiken. Anders moet je die files verplaatsen naar rawdata/ngs/

Reply:
Ook als FastQ bestanden niet in het door ons gewenste format zitten, zou ik ze in een submap per run/batch o.i.d. van rawdata/ngs/ neerzetten. Waar laat je ze anders zonder dat het een wildgroei aan locaties wordt; als je die data later nog een keer nodig hebt zoek je je anders suf. Het kan toch gewoon iets van
```../../../../../rawdata/ngs/${externalProject}/${afwijkendeNaam}.fq.gz
```
zijn.

Reply:
Daar wordt toch ook de symlink neergezet, die de afwijkende naar corrigeert naar ons format.